src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/BaseQualityRankSumTest.java

```diff
@@ -40,7 +40,12 @@ protected OptionalDouble getElementForRead(final GATKRead read, final int refLoc
 
     public static OptionalDouble getReadBaseQuality(final GATKRead read, final int refLoc) {
         Utils.nonNull(read);
-        return OptionalDouble.of(read.getBaseQuality(ReadUtils.getReadCoordinateForReferenceCoordinateUpToEndOfRead(read, refLoc, ReadUtils.ClippingTail.RIGHT_TAIL)));
+        final int baseOffset = ReadUtils.getReadCoordinateForReferenceCoordinateUpToEndOfRead(read, refLoc, ReadUtils.ClippingTail.RIGHT_TAIL);
```

Comment:
FYI This change is needed here because this annotations expect to get a proper value for each annotation qualifying read (the base call quality in this case) Now that some of these may not overlap with the variant we need to return a "no-value" for that read that is consequently ignored.